test/es-module/es-module.status

```diff
@@ -23,7 +23,6 @@ test-esm-loader-modulemap : SKIP
 test-esm-main-lookup : SKIP
 test-esm-named-exports : SKIP
 test-esm-namespace : SKIP
-test-esm-ok : SKIP
```

Comment:
Was this test just removed?

Reply:
Looks like it was moved in fddcd6253b237ca68430f44c7614bf52d57c4f97.